roles/openshift_master_facts/tasks/main.yml

```diff
@@ -49,6 +49,10 @@
       registry_url: "{{ oreg_url | default(None) }}"
       oauth_grant_method: "{{ openshift_master_oauth_grant_method | default(None) }}"
       sdn_cluster_network_cidr: "{{ osm_cluster_network_cidr | default(None) }}"
+      external_ip_network_cidrs:
+          {%- for openshift_master_external_ip_network_cidr in openshift_master_external_ip_network_cidrs %}
+          - "{{ openshift_master_external_ip_network_cidr }}"
+          {%- endfor %}
```

Comment:
The jinja loop sections of this will probably need to be unindented and the loop should reference the persisted fact.

```
       external_ip_network_cidrs:
{%- for external_ip_network_cidr in openshift.master.external_ip_network_cidrs %}
           - "{{ external_ip_network_cidr }}"
{%- endfor %}
```